pkg/datarepair/checker/checker.go

```diff
@@ -14,6 +14,8 @@ import (
 	"storj.io/storj/pkg/irreparabledb"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/pointerdb"
+	"storj.io/storj/pkg/statdb"
+	sdbpb "storj.io/storj/pkg/statdb/proto"
```

Comment:
Let's rename the package rather than using aliases. e.g. `statpb`

Reply:
Also since we are using direct requests, do we still need the proto file? @mobyvb

Reply:
having protos means we can scale things though doesn't it? If we get rid of it then the checker microservice just becomes the satellite

Reply:
@egonelbre the proto is necessary until we have another system for calling statdb functions. Right now it heavily depends on protobuf generated code.

Reply:
I would be very excited/happy to refactor statdb and get rid of the proto or make it use the pb package, but there are other things I am supposed to focus on at the moment